index.html

```diff
@@ -0,0 +1,157 @@
+                    <li><a href="about.html" class="fw-bold
+                        ">About</a></li>
+                    <li><a href="#programs-section" class="fw-bold
+                        ">Program</a></li>
+                    <li><a href="#join-section" class="fw-bold">Join</a></li>
+                    <li><a href="#featured-section" class="fw-bold
+                        "></a></li>
+                    <li class="last-list-item" class="fw-bold
+                    "><a href="#" class="fw-bold">AHMADI GYM</a></li>
+
+                </ul>
+            </nav>
+            </div>
+        </div>
+    </section>
+    <section class="first-page">
+        <header class="header-sect">
+            <a href="#"><i class="fas fa-bars fa-2xl" id="hamburger-menu"></i></a>
+        </header>
+        <article class="intro-sect">
+            <p class="heading-intro fw-bold">Stop Thinking Start Doing!</p>
+            <h2 class="fw-bold">Work-Sweat-Achieve</h2>
+            <h2 class="fw-bold"> Nothing Will Work Unless You Do</h2>
+            <h2 class="fw-bold">2023</h2>
+            <p class="intro-description">The Pain You Feel Today Will Be The Strength You Feel Tomorrow.</p>
+            <h3>2023.06.28(Wed)</h3>
+            <h4>@Kabul Afghanistan</h4>
+        </article>
+    </section>
+    <!-- Programs' Section -->
+    <section class="programs-sect" id="programs-section">
+        <h2 class="main-program">AHMADI GYM Program</h2>
+        <hr class="under-line">
+        <div class="program-activities">
+            <ul>
+                <li>
+                    <i class="fas fa-atom fa-2xl comments"></i>
+                    <h4>Trainings</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
+                </li>
+                <li>
+                    <i class="fas fa-burn fa-2xl comments"></i>
+                    <h4>Moments</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
+                </li>
+                <li>
+                    <i class="fas fa-brain fa-2xl comments"></i>
+                    <h4>Mindet</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
+                </li>
+                <li>
+                    <i class="fas fa-balance-scale fa-2xl comments"></i>
+                    <h4>Team</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
+                </li>
+                <li>
+                    <i class="fas fa-bahai fa-2xl comments"></i>
+                    <h4>Training Loop</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
+                </li>
+            </ul>
```

Comment:
- [ ] The cards of the main program section in the mobile view are not properly aligned in the center as seen in the original design, I would kindly suggest you work on it and be sure they are properly centered to follow the original design template.